src/cpp/shared/gui/dialog.h

```diff
@@ -5,9 +5,41 @@
 
 enum DIALOG_TYPE {
   DIALOG_OKAY = 1,
-  DIALOG_YES_NO = 2
+  DIALOG_YES_NO = 2,
+  DIALOG_OR = 1,
+  DIALOG_EMPTY = 0,
 };
 
+enum IMAGE_TYPE {
```

Comment:
This seems like an oddly general name. Are these used for anything other than dialogs?